dark/index.html

```diff
@@ -212,7 +212,7 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
      <section id="contact"> 
       <h1>CONTACT ME</h1>
       <h3>CONTACT <span class="text-info">ME</span></h3> 
-      <div class="formContainer my-3 bg-dark p-5 rounded-3">
+      <div class="formContainer bg-dark p-5 rounded-3">

```

Comment:
Don't remove this my-3 @Rishav1707 
```suggestion
      <div class="formContainer my-3 bg-dark p-5 rounded-3">
```

Reply:
@Rishav1707 pls undo these changes because it creates a gap between the text and the form I didn't knew it sorry for the inconvenience happened

Reply:
Do these changes @Rishav1707